core/ui/compose/preference/src/debug/kotlin/net/thunderbird/core/ui/compose/preference/ui/fake/FakePreferenceData.kt

```diff
@@ -42,9 +42,16 @@ internal object FakePreferenceData {
         options = choices,
     )
 
+    val switchPreference = PreferenceSetting.Switch(
+        id = "switch",
```

Comment:
These preferences are related to render UI elements, in this case it needs also a title and a description to be displayed to the user.